### DISCUSSION TITLE
Conditionally use separate pytest_plugin

### DISCUSSION THREAD

**Discussion by f00dog-x300** [pytest-dev/pytest#11061] [Q&A] [answered]
Hi all, 

I know creating a non-top-level conftest is on longer supported. I initially was using them to have separate tests that refer to files with fixtures on specific files. I was wondering if there is a way to conditionally say something like: 
- if the test is running on this directory use these sets of pytest_plugins

**Comment by RonnyPfannschmidt** [comment, marked as answer]
No,pluggy is currently not capable to support this,in fact all pytest plugins listed in a contest would always end up global

**Reply by f00dog-x300** [reply]
Thanks
